packages/checkout/widgets-lib/src/widgets/sale/functions/transformToClientConfig.test.ts

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { ClientConfig, SignPaymentTypes } from '../types';
+import {
+  ClientConfigResponse,
+  transformToClientConfig,
+} from './transformToClientConfig';
+
+describe('transformToClientConfig', () => {
+  it('transformToClientConfig', () => {
```

Comment:
good to describe what being tested 
`it('should return input object with camel case keys')`